Automattic/wp-calypso
# Theme Curation: LOTS & LITS Categories Design

Related to https://github.com/Automattic/dotcom-forge/issues/3370

Implemented the design decisions taken in the [parent task](https://github.com/Automattic/dotcom-forge/issues/3370) and in slack.


| Before | After |
|----------|----------|
|![Captura de Pantalla 2023-08-08 a las 13 18 23](https://github.com/Automattic/wp-calypso/assets/1989914/a2f0275a-9d22-4516-a801-46dc3de31c13)|![Captura de Pantalla 2023-08-08 a las 13 16 55](https://github.com/Automattic/wp-calypso/assets/1989914/10c83da5-b9f8-40ec-aca6-acbbb2118222)|
|![Captura de Pantalla 2023-08-08 a las 13 19 30](https://github.com/Automattic/wp-calypso/assets/1989914/5e8f398c-119e-4222-b0cd-514ba7bcb3e0)|![Captura de Pantalla 2023-08-08 a las 13 20 16](https://github.com/Automattic/wp-calypso/assets/1989914/e8201f7b-4ce3-41cc-a875-a032df4edcea)|

## Proposed Changes

* We've moved the pricing filter next to the search bar to make more space for the subject filters so that we can display more subjects.

## Testing Instructions



* Open the Calypso live link
* Navigate to the logged in Theme Showcase
* You should see the new design and it should work as expected. Test with different viewports.
* Navigate to the logged out Theme Showcase
* You should see the new design and it should work as expected. Test with different viewports.

## Pre-merge Checklist
